src-v2/Not/Not.Blazor/TM/NotTimer.razor

```diff
@@ -0,0 +1,63 @@
+﻿@using System.Timers
+@implements IDisposable
+
+<div style="@($"color:{@_color};")">
+   @_displayTime
+</div>
+
+@code {
+    [Parameter]
+    public string StartTime { get; set; } = default!;
+    [Parameter]
+    public double StopTimerAtTime { get; set; } = default!;
+
+    private Timer _timer = default!;
+    private MudTheme _theme = new MudTheme();
+    private TimeSpan _time = default!;
+    private string _displayTime = default!;
+    private string _color = default!;
+
+    protected override void OnParametersSet()
+    {
+        _time = TimeSpan.Parse(StartTime);
+        FormatTime();
+        _timer = new Timer(1000);
```

Comment:
Magic value.

Reply:
Also do you need to create a new timer every time a parameter is changed?

Reply:
Furthermore if we have a custom timer imlementation why not use that one for consistency?